ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -292,14 +294,14 @@ const Footer = ({
         }
     };
 
-    const goToPreviousStep = (activeStep, onBack) => {
+    const goToPreviousStep = (activeStep, onBack, errorHandler) => {
         // first reset validation state to default
         setIsFormValid(true);
         onBack();
         if (activeStep.id === "installation-review") {
             resetPartitioning().then(() => {
                 debug("resetPartitioning");
-            }, console.error);
+            }, errorHandler);
```

Comment:
I am not convinced this should be consiredered critical error. We should block going back in case this fails, but it's probable that the user can do something about it? Maybe try again?

Reply:
I guess at least on the live media they might just run the installer again ? I think @vojtechtrefny or @poncovka might want to chime in if the state of the storage backend makes it possible to attempt with the installation run without restarting Anaconda.

Reply:
> I am not convinced this should be consiredered critical error. We should block going back in case this fails, but it's probable that the user can do something about it? Maybe try again?

If resetPartitioning fails with exception, the installer is in undefined state IMO, so blocking going back would not help.


> I guess at least on the live media they might just run the installer again ? I think @vojtechtrefny or @poncovka might want to chime in if the state of the storage backend makes it possible to attempt with the installation run without restarting Anaconda.

Yes, and restarting the installation is a consequence of Critical Error I think. Here the hint (another context element I am removing in later commit because I didn't find a use yet, although I wasn't looking too hard) could be useful. OTOH we have just Reboot option in the dialog (and I think it is on purpose at this point, we want to avoid restarting anaconda after critical error because we are not sure if it would work) so we would also need some logic allowing just Quit.
In any case something to be handled separately as a follow-up I think.
And I can remove the patch from the set if you prefer.

Reply:
> About the possibility to recover from unexpected errors, I don't recommend to do that. They are bugs that need to be fixed. The recovery could lead to weirder bugs and errors that can be difficult to debug and fix.

Totally agreed & I will also add that another reason to stop when we hit a bug is to possibly prevent some irreversible issues caused by the bug. Not being able to install sucks, machine that does not boot or lost data sucks more.